src/main/scala/org/kframework/minikore/implementation/MiniKore.scala

```diff
@@ -1,122 +1,163 @@
 package org.kframework.minikore.implementation
 
-import org.kframework.minikore.interfaces.{build, pattern => i}
+import org.kframework.minikore.interfaces.{build, pattern => p}
 
 import scala.collection._
 
-/** Algebraic data type of MiniKore. */
+/** Algebraic data type of MiniKore. */ // TODO: should rename to "pattern"
 object MiniKore {
 
   /** A collection of classes that serve as the default implementation of the [[org.kframework.minikore.interfaces.pattern]] **/
 
-  type Attributes = Seq[i.Pattern]
+  // Pattern Implementations
+  // =======================
```

Comment:
This diff is not readable. Could you make it to have a more meaningful diff? Otherwise, please briefly explain what was changed.